src/Order/Base.php

```diff
@@ -540,7 +547,6 @@ public function delete_meta_value( $order_id ) {
 		$this->delete_label( $saved_data );
 		unset( $saved_data['backend'] );
 		unset( $saved_data['labels'] );
-		unset( $saved_data['barcode'] );
```

Comment:
@iyut  we need to delete the barcodes after deleting the label  regarding to [this requirement](https://app.clickup.com/t/861mpdmz6)

Reply:
Thanks for reviewing this PR @Abdalsalaam , so are you confirming that we dont need to saved the barcode for 7 days? Because [this code](https://github.com/Progressus-io/postnl-for-woocommerce/blob/a6a753e0271616d80cafaf62584936103743ee51/src/Order/Base.php#L620-L626) seems we need to use the saved barcode at least within 7 days.

Reply:
@iyut yes we need to save the barcode for 7 days, so if the label creation failed it will use the same barcode on the next try, but deleting the label should delete the barcode as well.